include/aws/common/array_list.inl

```diff
@@ -58,8 +61,9 @@ void aws_array_list_init_static(
     assert(item_size);
 
     list->alloc = NULL;
-
-    list->current_size = item_count * item_size;
+    if(!aws_mul_u64_checked(item_count, item_size, &list->current_size)) {
+        return AWS_OP_ERR;
```

Comment:
This function has a return type of void. Did you compile this code locally?

Reply:
Sorry - I forgot to mark this "review only"

Reply:
Fixed.